frontend/src/components/NavBar/PrintImage/image.tsx

```diff
@@ -434,15 +434,20 @@ function DownloadImage({ classes, open, handleClose }: DownloadImageProps) {
                       {titleText}
                     </div>
                   )}
-                  {footerTextSize > 0 && footerText && (
+                  {footerTextSize > 0 && (footerText || dateText) && (
                     <div
                       ref={footerRef}
                       className={classes.footerOverlay}
                       style={{
                         fontSize: `${footerTextSize}px`,
                       }}
                     >
-                      <div style={{ padding: '8px' }}>{footerText}</div>
+                      {footerText && (
+                        <div style={{ padding: '8px' }}>{footerText}</div>
+                      )}
```

Comment:
The double padding between the footerText and the dateText feels a bit unecessary

<img width="814" alt="Screenshot 2024-05-13 at 12 32 55 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/ec180113-3ea5-4d1b-9e79-09cfa670a4b6">

Reply:
Fixed